tools/swift-inspect/Sources/SwiftInspectLinux/AuxVec.swift

```diff
@@ -0,0 +1,73 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import LinuxSystemHeaders
+
+internal class AuxVec {
+  enum Error: Swift.Error { case FileReadFailure(_ filePath: String) }
+
+  // enum values must match the constants defined in usr/include/linux/auxv.h
+  enum Tag: UInt64 {
```

Comment:
Can we use a module map to import `linux/auxv.h` instead? This seems unnecessarily fragile.

Reply:
Sure, I'll drop the `enum`.